x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/integration/DocumentAndFieldLevelSecurityTests.java

```diff
@@ -144,6 +149,38 @@ public void testSimpleQuery() {
         assertThat(response.getHits().getAt(1).getSourceAsMap().get("field2").toString(), equalTo("value2"));
     }
 
+    public void testUpdatesAreRejected() {
+        String indexName = randomFrom("<test-{2015.05.05||+1d}>", "test");
```

Comment:
Why would we randomise on this rather than just have 2 tests?
It feels like having a permanent test with datemath would be a good thing.

Reply:
I've made it a for-each to replace the randomisation.

I would've opted for a separate test myself, but since this is an integration test, I've accommodated the requirement to not have exhaustive ITs by using randomisation.